Website/src/rendering/src/assets/css/main.css

```diff
@@ -91,6 +91,7 @@
 @import "./components/Speakers/SpeakerInformationPageHero.css";
 @import "./components/Speakers/SpeakerList.css";
 @import "./components/Speakers/SpeakersGrid.css";
+@import "./components/Speakers/FeaturedSpeakers.css";
```

Comment:
Please order the imports alphabetically by folder like the rest of the imports. Thanks.